engines/rbd.c

```diff
@@ -529,6 +532,11 @@ static int fio_rbd_queue(struct thread_data *td, struct io_u *io_u)
 static int fio_rbd_init(struct thread_data *td)
 {
 	int r;
+	struct rbd_data *rbd;
+	rbd = (struct rbd_data*)(td->io_ops_data);
+	if(rbd && rbd->connected) {
```

Comment:
seems impossible to be NULL. Add if(rbd &&...) just want to make code analysis tool clean. Do you prefer to remove this check？